### DISCUSSION TITLE
How to merge cells in a row

### DISCUSSION THREAD

**Discussion by soundstep** [charmbracelet/lipgloss#407] [Ideas]
Hi,
How would I merge the cells on the last row so that they take up all the width of the table?
```sh
t := table.New()
t.Row("col 1", "col 2", "col 3")
t.Row("col 1", "col 2", "col 3")
t.Row("A new very long row taking the table width")
fmt.Println(t.Render())
```

**Comment by bashbunni** [comment]
Hey @soundstep we don't currently support that feature. 

![image](https://github.com/user-attachments/assets/3e13574d-f2dc-4f18-bec5-f849dc9712d7)

You would want that final row to appear as only one cell right?

**Reply by soundstep** [reply]
Yes something like
```
|---------------------------|
| col 1  | col 2 | col 3    |
|---------------------------|
| val 1  | val2  | val 3    |
|---------------------------|
| a really really long cell |
|---------------------------|
```
I didn't realise this wasn't supported.

**Comment by bashbunni** [comment]
Just changed this to an idea in case others are interested in this feature too, then we can gauge interest with this discussion :)
